### PR TITLE
Rename to FastTrackML

### PR DESCRIPTION
**Pull request by jgiannuzzi**
This PR renames everything FastTrackML.

* The Docker Hub repo has been recreated as `gresearch/fasttrackml`
  * `vars.DOCKER_REPO` has been updated
  * `skopeo copy` was used to copy all the existing tags
* The ghcr.io repo will be recreated as `ghcr.io/g-research/fasttrackml`
  * `skopeo copy` will be used after the first image gets published
* The (temporary) logo has been updated in the MLFlow UI
* All docs have been updated
* The Go module path has been updated to `github.com/G-Research/fasttrackml`
* The GitHub repo has been renamed to `G-Research/fasttrackml`
* The command has been renamed to `fml`
* The environment variable prefix has been renamed from `FASTTRACK_` to `FML_`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # IDE
 .idea
 
+fml
 fasttrack
 fasttrack.db*
```
Comment (DSuhinin) [inline .gitignore:6]: do we need the old one?

Comment (DSuhinin) [reply .gitignore:6]: also, maybe we can add more comments here like I did for IDE just do better understand the meaning of this files included into `gitignore`. what do you think?

**Review by DSuhinin — DISMISSED** [review]
just a few small comments.

**Review by fabiovincenzi — APPROVED** [review]

**Review by fabiovincenzi — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jgiannuzzi) [reply .gitignore:6]: I kept the old one for compatibility, but actually yes, let's just remove it!

I will also add some comment to `.gitignore`, good point!

**Review by DSuhinin — APPROVED** [review]
good to go!

**Review by fabiovincenzi — APPROVED** [review]

**Review by m4rs-mt — APPROVED** [review]
LGTM so far; except that I cannot verify the correctness of the `custom.patch` file.

**Pull request merged**